### DISCUSSION TITLE
How can I override message that shows in empty table?

### DISCUSSION THREAD

**Discussion by Dozez13** [quasarframework/quasar#9393] [General - Components / Directives / etc] [answered]
When data is empty table show "No matching records found", how can I change that message?Thank for help.

**Comment by metalsadman** [comment, marked as answer]
> When data is empty table show "No matching records found", how can I change that message?Thank for help.

use `no-data-label` prop, ie. `no-data-label="Your custom message..."`.

**Reply by Dozez13** [reply]
Thanks for reply
